### PR TITLE
hwdb: add missing Group()

### PR DESCRIPTION
**Pull request by yuwata**
C.f. https://github.com/pyparsing/pyparsing/wiki/Parser-Debugging-and-Diagnostics#__diag__warn_ungrouped_named_tokens_in_collection

### CODE REVIEW

File: hwdb.d/parse_hwdb.py [hunk 1]
```diff
@@ -1,27 +1,5 @@
...
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-# SOFTWARE.
```
Comment (keszybz) [inline hwdb.d/parse_hwdb.py]: The idea is that this file is separately-distributable from the rest of systemd. That it why it has a different license than our other files, and that is why it also has the boilerplate [to specify the license through an explicit text and not just the SPDX tag]. (For files under [L]GPL we keep the license text in LICENSE.* files in the root of the repo.) Please keep the boilerplate.

**Review by keszybz — CHANGES_REQUESTED** [review]
Could you move the PR comment into the commit? I assume it's to fix a warning with a new version, but it's not immediately clear.

File: hwdb.d/parse_hwdb.py (hunk 1 above)
Comment (yuwata) [reply hwdb.d/parse_hwdb.py]: Oh, yeah, I forgot that. Thanks. Dropped.

**Comment by yuwata** [comment]
Updated. PTAL.

**Pull request merged**